tests/github/fair_software/badge/test_checker_with_reasonable_user_config.py

```diff
@@ -0,0 +1,123 @@
+import pytest
+from requests_mock import Mocker
+from howfairis import Checker
+from howfairis import Repo
+from tests.contracts.checker import Contract
+
+
+def get_mocked_checker(user_config_filename):
+    repo = Repo("https://github.com/fair-software/badge")
+    return Checker(repo, user_config_filename=user_config_filename)
+
+
+@pytest.fixture
+def user_config(tmp_path):
+    config = tmp_path / "howfairis.yml"
+    text = """skip_repository_checks_reason: dunno1
+skip_license_checks_reason: dunno2
+skip_registry_checks_reason: dunno3
+skip_citation_checks_reason: dunno4
+skip_checklist_checks_reason: dunno5
+"""
+    config.write_text(text)
+    yield config
```

Comment:
Probably this fixture can go in `conftest.py`. While this introduces magic, it is the standard pattern

Reply:
Well then I cannot use same name for fixture in test_checker_with_noreason_user_config.py. To me this is a fixture only used in this file. While mocker is used in all tests in the directory.

Reply:
ok